bin/version-manager.js

```diff
@@ -114,16 +114,18 @@ function resolveGlobs(globs, cb) {
 
 function run(files) {
   // Clean up the files we'll be running.
-  // TODO: Another case to update with `Set` once Node v0.8 and v0.10 are dropped.
-  files = files.sort().map(function(p) {
-    return path.resolve(p)
+  let filePaths = new Set()
+  files.sort().forEach((file) => {
+    filePaths.add(path.resolve(file))
   })
-  var directories = Object.keys(files.reduce(function(dirs, p) {
-    dirs[path.dirname(p)] = true
-    return dirs
-  }, {}))
 
-  var mode = cmd.major ? 'major' : cmd.patch ? 'patch' : 'minor'
+  let directories = new Set()
+  filePaths.forEach((filePath) => {
+    directories.add(path.dirname(filePath))
+  })
+  directories = Array.from(directories)
```

Comment:
Looks like we are using Set for de-duplication.

Minor: remapping the type of directories is a bit awkward. Prob my type-safe background kicking in.

Reply:
Ya it was a bit awkward. I would've left it a Set, but later on some code called a function on it that would only work with arrays..

Reply:
yeah. mostly i just don't like change types on an individual variable vs introducing a new variable so usage is consistent. but minor given the isolation of this usage.